drivers/Aqara/aqara-lock/src/init.lua

```diff
@@ -51,6 +52,23 @@ local function remoteControlShow(device)
   end
 end
 
+local function comp_supported_alarm_values(last_alarm_values)
+  if #last_alarm_values~=#SUPPORTED_ALARM_VALUES then return false end
```

Comment:
`#last_alarm_values` will error the device init handler if the device does not have the field set.

Reply:
A nil check here is probably warranted.

Reply:
Okay, I'll add a nil check.